install.sh

```diff
@@ -336,7 +336,7 @@ parse_flags ()
     while :; do
         case $1 in
             --develop)
-                develop_build='1'
```

Comment:
Is this variable needed? It wasn't referenced anywhere.

Is the `--develop` flag needed?

Reply:
No, it isn't anymore. Command line flag still exists for backwards compatibility.